### ISSUE TITLE
Cannot disable automatically reading

### ISSUE THREAD

**Issue by westor7** [fastfinge/adispeak#20]
Hello,

I used /sapioff but didn't stopped from speaking, and by default when i load the dll it has that feature enabled, it could be better to have that by default off.

- Thanks!

**Comment by fastfinge** [comment]
What screen reader are you using?  Adispeak will always revert to SAPI if it can't detect your currently running screen reader.  If you're not running a screen reader, speaking with sapi is the entire point of the DLL, so there's no way to disable it.  If we're failing to detect your screen reader, I can try and fix that though.

**Comment by westor7** [comment]
What do you mean "screen reader" ?

**Comment by fastfinge** [comment]
The entire purpose of this dll is to make IRC accessible to those of us who are blind, by interfacing with one of the programs we use to read our computer screens. Myself, I run NVDA (https://www.nvaccess.org/) to read the computer screen to me.  If the dll can't find NVDA, or a similar program we support (Jaws for Windows, Window Eyes, System Access, etc) it will fall back on reading everything by SAPI. You can't turn SAPI off unless you're running a supported screen reading program.  That behavior is intended, and not a bug.

**Comment by westor7** [comment]
Ah ok i didn't know that this DLL was created for that purpose, apologize
